lib/html2markdown/converter.rb

```diff
@@ -51,11 +51,17 @@ def wrap_node(node,contents=nil)
       when 'strong'
         result << "**#{contents}**\n"
       when 'h1'
-        result << "##{contents}\n"
+        result << "# #{contents}\n"
       when 'h2'
-        result << "###{contents}\n"
+        result << "## #{contents}\n"
       when 'h3'
-        result << "####{contents}\n"
+        result << "### #{contents}\n"
+      when 'h4'
+        result << "#### #{contents}\n"
+      when 'h5'
+        result << "##### #{contents}\n"
+      when 'h6'
+        result << "###### #{contents}\n"
```

Comment:
I added handling for h4-h6, since that's standard markdown formatting, and just added a space between the heading markers and the heading content, and did the same with `li`'s